doc/doc.md

```diff
@@ -22,6 +22,8 @@ With `delete` format the data is transformed as normal but the output will be a
 
 If no output file is specified the transformed data will be written to stdout.
 
+The `--abort-on-error` flag will prevent any output being generated but the mapper will still process the whole input (so as to find all errors) then exit with an error status.
```

Comment:
Worth adding the proviso that no output is generated *if* errors are encountered.